### DISCUSSION TITLE
No valid __init__.py

### DISCUSSION THREAD

**Discussion by pedroNMR9** [noDRM/DeDRM_tools#112] [Q&A] [answered]
Hi all,

I hope I am not asking to pure newbie question : 
When I try to install the plugin on Calbre 6.1, I always end up with an InvalidPlugin Error : 
<img width="495" alt="image" src="https://user-images.githubusercontent.com/109901799/180647027-eb4450c1-6014-4363-9067-102941f09ec5.png">

What do I do wrong? Is this a known issue?

BR,

Pierre

**Comment by ElleKayEm** [comment, marked as answer]
You're trying to load the wrong file.  Go here: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

Download DeDRM_tools_10.0.3.zip.  Unzip it and load DeDRM_plugin.zip.

**Reply by pedroNMR9** [reply]
. I knew it must be some newbie stuff. It worked. 
Thanks!!!
